packages/README.md

```diff
@@ -0,0 +1,14 @@
+# Cal-ITP Python Packages
+
+We publish calitp-data-analysis and calitp-data-infra, two Python packages that enable work elsewhere in the repository. Each is maintained and deployed the same way.
+
+## Testing Changes
+
+Each package should pass mypy and other static checkers, and each has a small
+number of tests. These checks are executed in GitHub Actions when a PR is opened.
```

Comment:
Is it worth linking to the relevant GH Actions?

Reply:
Addressed in [ac52c02](https://github.com/cal-itp/data-infra/pull/3156/commits/ac52c0287d433386871beca0c45af14ede6b1f84)